pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-    "globus-sdk==3.35.0",
+    "globus-sdk==3.37.0",
```

Comment:
I just pushed through a general dependency update (today? yesterday? it's a blur), so this will probably be an annoying conflict. It's the same change for this line though, so :crossed_fingers: `git` could handle it gracefully.

Reply:
Sure.

Reply:
Sorry, I misread this as you asking me to update to 3.37.0...
But then I went to update to 3.37.0 and I was already on it!

Looks like no merge conflicts between this and upstream-main?